include/trace/events/sof_intel.h

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+//
+// This file is provided under a dual BSD/GPLv2 license.  When using or
+// redistributing this file, you may do so under either license.
+//
+// Copyright(c) 2022 Intel Corporation. All rights reserved.
+//
+// Author: Noah Klayman <noah.klayman@intel.com>
+//
+
+#undef TRACE_SYSTEM
```

Comment:
why do I see changes here even though this file was already added in the previous patch? Something odd is going on with GitHub?

Reply:
> why do I see changes here even though this file was already added in the previous patch? Something odd is going on with GitHub?

answering to my own question: different file! sof.h v. sof_intel.h.